hanjaelee/services/userService.js

```diff
@@ -1,17 +1,24 @@
+const bcrypt = require('bcrypt')
+
 const userDao = require('../models/userDao.js')
 
 const signup = async (name, email, password, profileImage) => {
+  const hashedPassword = await makeHashedPassword(password, 12)
```

Comment:
한재님 ! 암호화시 cost factor의 경우 상수로 존재하기 때문에 인자에 곧바로 숫자 값을 넣는 것 보다.
상수로 선언하여 사용하는 것이 좋습니다!
```suggestion
const COST_FACTOR = 12;
  const hashedPassword = await makeHashedPassword(password, COST_FACTOR)
```